Makefile

```diff
@@ -14,6 +14,9 @@ WLRM_FOLDER       = stable-$(shell echo ${NGINX_VERSION} | sed 's/\.//g')
 GOMPLATE_VERISON  = 3.11.7
 COMMIT_SHA        ?= git-$(shell git rev-parse --short HEAD)
 
+ALLURE_ENDPOINT ?= https://allure.wallarm.com
```

Comment:
please, place it below line 35 `.EXPORT_ALL_VARIABLES:`

Reply:
moved